src/Apps/Settings/Routes/SavedSearchAlerts/SavedSearchAlertsApp.tsx

```diff
@@ -48,19 +50,35 @@ export const SavedSearchAlertsApp: React.FC<SavedSearchAlertsAppProps> = ({
   me,
   relay,
 }) => {
-  const [
-    editAlertEntity,
-    setEditAlertEntity,
-  ] = useState<EditAlertEntity | null>(null)
   const { relayEnvironment } = useSystemContext()
   const { sendToast } = useToasts()
   const { trackEvent } = useTracking()
+  const { silentPush } = useRouter()
   const [showDeleteModal, setShowDeleteModal] = useState(false)
   const [sort, setSort] = useState("ENABLED_AT_DESC")
   const [loading, setLoading] = useState(false)
   const alerts = extractNodes(me.alertsConnection)
+  const isMobile = __internal__useMatchMedia(THEME.mediaQueries.xs)
```

Comment:
Unfortunately we can't use this, and that's why its marked as `__internal__`. Not SSR safe. Of course, if the code-path never hits ssr its fine, but that can't be promised in the future.

Reply:
thanks!

Reply:
Thank you 🙏